scripts/protocgen.sh

```diff
@@ -3,7 +3,7 @@
 set -eo pipefail
 
 # get protoc executions
-go get github.com/regen-network/cosmos-proto/protoc-gen-gocosmos 2>/dev/null
+# go get github.com/regen-network/cosmos-proto/protoc-gen-gocosmos 2>/dev/null
```

Comment:
why is this commented?

Reply:
it was failing for me to run `make proto-gen` and since the go get bellow 

`# go get github.com/cosmos/cosmos-sdk 2>/dev/null`

was already commented, I just commented it out instead of deleting